packages/forms-web-app/src/app.js

```diff
@@ -90,6 +92,7 @@ app.use(isProjectClosed);
 // View Engine
 app.set('view engine', 'njk');
 
+app.use(plannedOutage);
```

Comment:
I wonder if the showOutagePage guard should be here.  It would avoid that check for every request.
We also have to restart every time we activate this so can assume web server gets restarted to activate this.

Reply:
good point

Reply:
Isn't the config only built once from the environment variables? If so this issue would happen regardless